composer.json

```diff
@@ -17,6 +17,11 @@
             "ZendService\\Amazon\\": "library/"
         }
     },
+    "autoload-dev": {
+        "psr-0": {
+            "ZendServiceTest\\Amazon\\": "tests/ZendServiceTest/Amazon/"
```

Comment:
I'm sure PSR-0 and PSR-4 are mixed in this syntax. PSR-0 syntax should be simmetric to "require"